doc/AST_FORMAT.md

```diff
@@ -322,6 +322,20 @@ Format:
  ~~~~~ expression
 ~~~
 
+
+### Endless (2.6)
+
+Can be inclusive or exclusive.
+
+Format:
+
+~~~
+(irange (int 1) nil)
+"1.."
+  ~~ operator
+ ~~~ expression
```

Comment:
I try to list all forms explicitly in this document, i.e. both `irange` and `erange`.